pep-0484.txt

```diff
@@ -610,6 +612,24 @@ the runtime class of the objects created by instantiating them doesn't
 record the distinction. This behavior is called "type erasure"; it is
 common practice in languages with generics (e.g. Java, TypeScript).
 
+Using generic classes (parameterized or not) to access attributes will result
+in both type check failure and runtime error. Outside the class definition
```

Comment:
Ah right. I misunderstood your comment that [it's fixed](https://github.com/python/mypy/issues/2878#issuecomment-282302705) in `typing`. I thought you meant it now causes runtime error, but actually what you meant is that it no longer returns different values depending on how it's accessed.